tests/tests_scripts/test_examples.py

```diff
@@ -11,7 +11,7 @@
 
 
 BACKEND = os.environ.get("GEOMSTATS_BACKEND", "numpy")
-ALL_BACKENDS = ["numpy", "autograd", "pytorch", "tensorflow"]
+ALL_BACKENDS = ["numpy", "autograd", "pytorch"]
 AUTODIFF_BACKENDS = ALL_BACKENDS[1:]
 NP_LIKE_BACKENDS = ALL_BACKENDS[:2]
```

Comment:
NP_LIKE_BACKENDS is equal to ALL_BACKENDS now: remove?

Reply:
I don't think this is quite right.